extensions/wikia/Email/templates/avatarLayout.mustache

```diff
@@ -1,6 +1,6 @@
 <div class="greeting">
 	<p>{{salutation}}</p>
-	<p>{{{summary}}}</p>
+	<p><strong>{{{summary}}}</strong></p>
```

Comment:
Okay. Since my branch is the one that pulls out this greeting into a partial I just added the CSS change to my branch.

Here's the commit: https://github.com/Wikia/app/commit/d0a029b1940f7c7c46bc2f1f72bd8264e9928c1c

I think when we're resolving the conflicts between these 2 we can just resolve to the changes in my branch (basically making sure we have [this change](https://github.com/Wikia/app/pull/7426/files#diff-b467f3bb9d5d158bafee32774e7a8506R1), and [this file](https://github.com/Wikia/app/pull/7426/files#diff-3035bd2d79079302c4084f10c8d10e9bR1)).

@mklucsarits, let me know if you think I should be using a different selector or something there.